apps/backend/src/routes/chip/leaderboard_entry.ts

```diff
@@ -17,46 +17,6 @@ import { Controller } from "@/lib/controller";
 const router = express.Router();
 const controller = new Controller();
 
-/**
- * @route GET /api/chip/get_leaderboard_entry
- * @desc Gets the leaderboard entry for a user
- */
-router.get(
-  "/get_leaderboard_entry",
-  async (
-    req: Request<{}, {}, GetLeaderboardEntryRequest>,
-    res: Response<LeaderboardEntry | ErrorResponse>
-  ) => {
-    try {
-      const validatedData = GetLeaderboardEntryRequestSchema.parse(req.query);
-      const { authToken, chipIssuer } = validatedData;
-
-      // Fetch user by auth token
-      const user = await controller.GetUserByAuthToken(authToken);
-
-      if (!user) {
-        return res.status(401).json({ error: "Invalid auth token" });
-      }
-
-      // Get leaderboard entry
-      const entry = await controller.GetLeaderboardEntry(
-        user.username,
-        chipIssuer
-      );
-
-      if (entry) {
-        return res.status(200).json(entry);
-      }
-
-      throw new Error("Failed to get leaderboard entry");
-    } catch (error) {
-      return res.status(500).json({
-        error: errorToString(error),
-      });
-    }
-  }
-);
```

Comment:
Removing because it's unused? Kind of too bad, I believe at one point I used it for testing purposes and it was useful.

Reply:
yes we have it in git history, i think we can always add it if needed but i was confused by its existence